inc/elementor/js/ang-typography.js

```diff
@@ -23,32 +23,44 @@ jQuery( document ).ready( function() {
 	} );
 
 	function addPageStyleSettings( groups ) {
-		const PageStyles = {
-			name: 'ang_styles',
-			actions: [
-				{
-					name: 'page_styles',
-					title: ANG_Action.translate.pageStyles,
-					callback: function() {
-						analog.redirectToSection();
-					},
-				},
-				{
-					name: 'theme_style',
-					title: elementor.translate( 'Theme Style' ),
-					callback: analog.openThemeStyles,
+		const actions = [
+			{
+				name: 'page_styles',
+				title: ANG_Action.translate.pageStyles,
+				callback: function() {
+					analog.redirectToSection();
 				},
+			},
+			{
+				name: 'theme_style',
+				title: elementor.translate( 'Theme Style' ),
+				callback: analog.openThemeStyles,
+			},
+		];
+
+		if ( parseInt( ANG_Typo.has_sk_colors ) ) {
+			actions.push(
 				{
 					name: 'global_colors',
 					title: elementor.translate( 'Edit Style Kit Colors' ),
 					callback: analog.openGlobalColors,
-				},
+				}
+			);
+		}
+
+		if ( parseInt( ANG_Typo.has_sk_fonts ) ) {
```

Comment:
@lushkant shouldn't this evaluate for `Boolean`?

Reply:
You see that only returns 1 or 0 in string so this works, quite efficiently. I've seen this pattern in checkbox validations so thought why not.

But if you strictly want it to be validated as bool, we can do that too?

Reply:
Could be! But I don't see a reason to treat expected Boolean value as Int?

Reply:
alright, I'll do that!